client/src/components/HostViews/Respond.vue

```diff
@@ -1,72 +1,50 @@
 <template>
-  <div> 
-    <div style="position: absolute; top: 10%; left: 2%">
-      <Clock />
-    </div> 
-    <div style="position: absolute; top: 10%; right: 2%">
-      <Clock />
-    </div> 
-    <Prompt 
-      :words="words"
-      @prompt-intro-over="preIntroResponseStyles = ''"
-    />
-    <div class="center">
-      <div :style="`transition: 3s; width: 60%; ${preIntroResponseStyles}`">
-        <v-row align="center" justify="center">
-          <v-col
-            v-for="player in playerList" 
-            :key="player.clientId"
-            cols="6"
-          >
-            <ResponseCard
-              :playerName="player.name"
-              :hasResponded="respondents.includes(player.name)"
-            />
-          </v-col>
-        </v-row>
+  <div class="background-matte">
+    <header class="page-header center">
+      <div 
+        class="ma-3"
+        style="position: absolute; top: 0; right: 0;"
+      >
+        <Clock />
+      </div>
+      <h2 class="sub-title">Rhyme These {{ words.length }} Words with Eachother</h2>
```

Comment:
I agree with the word `Rhyme` being upper case but lowercase the rest please and add an exclamation mark.

Reply:
im not sure if lowercase is better because it is part of the title. take a look and if you still believe it should be changed i will change it
<img width="358" alt="Screen Shot 2022-11-24 at 4 00 34 PM" src="https://user-images.githubusercontent.com/76519301/203862131-fd7d085f-c5b7-42ae-b3ab-1b4f88dff7d7.png">

Reply:
Yeah ok. I see why you did it that way. Fine by me then